lib/queries.js

```diff
@@ -207,17 +220,14 @@ export const GetMySubscriptions = gql`
   }
 `;
 
-//done
 export const DeleteMySubscription = gql`
-  mutation DeleteMySubscription($subscriptionId: String!) {
+  mutation DeleteMySubscription($subscriptionId: Float!) {
```

Comment:
`Int!` not `Float!`. I'd suggest using `String!` in any case.

Reply:
I actually got an error message when it was `Int!` that said it was expecting a type of `Float!`, but I'll try `String!`

Reply:
Oh, just read your next comment down and saw why. Shouldn't be typing subscriptionId as "number"